### ISSUE TITLE
WCAG Compliance: 1.4.3 Contrast

### ISSUE THREAD

**Issue by owenslo** [RunestoneInteractive/RunestoneComponents#956]
We are verifying and improving Runestone's level of WCAG compliance to Level AA, our findings are summarized on a Google Doc here. This is for guideline 1.4.3
1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)
Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.

In order to meet this guideline, we need to make sure that the text has a contrast ratio of at least 4.5:1, except for the what text can be excluded.
Part of the Berea College Open Source Class @clarkh97 @owenslo @EzraSkwarka
